plugin/loader/loader.go

```diff
@@ -266,7 +266,7 @@ func (loader *PluginLoader) Close() error {
 	started := loader.started
 	loader.started = nil
 	for _, pl := range started {
-		if pl, ok := pl.(plugin.PluginDaemon); ok {
+		if pl, ok := pl.(plugin.Closer); ok {
```

Comment:
`plugin.Closer` feels a bit overkill. What about just type-asserting `pl.(io.Closer)`?

Reply:
I'm ok with that if you are, I just though the semantic was a bit off (it's not really an I/O).

Reply:
`io.Closer` is really just the standard "closer" at this point.